aruco_pose_estimation/scripts/aruco_node.py

```
@@ -372,7 +352,71 @@ def initialize_parameters(self):
+        pose_array = PoseArray()
+        if self.camera_frame == "":
+            markers.header.frame_id = self.info_msg.header.frame_id
+            pose_array.header.frame_id = self.info_msg.header.frame_id
+        else:
+            markers.header.frame_id = self.camera_frame
+            pose_array.header.frame_id = self.camera_frame
+
+        T_marker_cam = TransformStamped()
+
+        try:
+            frame, pose_array, markers = pose_estimation(rgb_frame=self.cv_image, depth_frame=None,
```

Ruff (0.14.14)

[warning] Unpacked variable `frame` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)